__tests__/sinopiaApi.test.js

```diff
@@ -401,3 +402,25 @@ describe("putUserHistory", () => {
     )
   })
 })
+
+describe("postTransfer", () => {
+  describe("success", () => {
```

Comment:
I'm only looking at the github diffs of this file -- would we want a test for failure?

Reply:
No, it intentionally doesn't handle a failure. (The failure is passed on to the calling code.)